tests/Unit/Adapter/Configuration/PhpParametersTest.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Unit\Adapter\Configuration;
+
+use InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Adapter\Configuration\PhpParameters;
+
+class PhpParametersTest extends TestCase
+{
+    /** @var string */
+    protected $parametersSampleFile;
+    /** @var string */
+    protected $parametersSampleFileContent;
+
+    protected $sampleParams = [
+        'parameters' => [
+            'a' => '127.0.0.1',
+            'b' => '',
+            'c_d' => '1234',
+        ],
+    ];
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function setUp(): void
+    {
+        $this->parametersSampleFile = dirname(__DIR__, 2) . '/Resources/config/params.php';
+        $this->parametersSampleFileContent = file_get_contents($this->parametersSampleFile);
+    }
+
+    protected function tearDown(): void
```

Comment:
```suggestion
    /**
     * {@inheritDoc}
     */
    protected function tearDown(): void
```
I think you have to add this PHPDoc

Reply:
What benefits does this bring? (sincere question)

Reply:
To keep coherence with the `setUp`! If you don't want to put on this method, I think you have to remove this on the `setUp` method